geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/iterators/KryoDtgAgeOffIterator.scala

```diff
@@ -0,0 +1,68 @@
+/***********************************************************************
+ * Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.accumulo.iterators
+
+
+import org.apache.accumulo.core.data.{Key, Value}
+import org.apache.accumulo.core.iterators.{IteratorEnvironment, SortedKeyValueIterator}
+import org.joda.time.format.{ISOPeriodFormat, PeriodFormatter}
+import org.joda.time.{DateTime, DateTimeZone}
+import org.locationtech.geomesa.accumulo.iterators.KryoDtgAgeOffIterator._
+import org.locationtech.geomesa.index.iterators.IteratorCache
+import org.opengis.feature.simple.SimpleFeature
+
+/**
+  * Age off data based on the dtg value stored in the SimpleFeature
+  */
+class KryoDtgAgeOffIterator extends KryoLazyAgeOffFilter {
```

Comment:
don't need to change, but i've been trying to get away from naming everything 'KryoFooIterator' - since all our data is kryo now, it's kind of redundant.